esp/esp/program/modules/forms/admincore.py

```diff
@@ -32,13 +33,21 @@ def __init__(self, program, *args, **kwargs):
     def load_data(self):
         lunch_timeslots = Event.objects.filter(meeting_times__parent_class__parent_program=self.program, meeting_times__parent_class__category__category='Lunch').distinct()
         self.initial['timeslots'] = lunch_timeslots.values_list('id', flat=True)
+        should_generate_constraints = True
+        if Tag.getTag('no_lunch_constraints'):
```

Comment:
Since lunch constraints are a policy that's likely to remain stable for a given chapter, I was trying to make it possible to set a default, but it's just one checkbox so you're right that it's probably unnecessary.  I will get rid of it.